app/controllers/investigations/ownership_controller.rb

```diff
@@ -79,10 +79,18 @@ def form
   end
 
   def get_potential_assignees
-    @selectable_users = [@investigation.owner_user, current_user].uniq.compact
-    @team_members = current_user.team.users.active.includes(:team)
-    @selectable_teams = [current_user.team, Team.get_visible_teams(current_user), @investigation.owner_team].flatten.uniq.compact
+    @team_members = current_user.team.users.active.includes(:team) - [@investigation.owner_user, current_user]
     @other_teams = Team.not_deleted
     @other_users = User.active.includes(:team)
+    @other_teams_added_to_case = @investigation.non_owner_teams_with_access
+    @default_opss_teams = (default_opss_teams - [@investigation.owner, current_user.team] - @other_teams_added_to_case)
+  end
+
+  def default_opss_teams
+    if current_user.is_opss?
+      Team.get_visible_teams(current_user)
+    else
+      Team.where(name: "OPSS Incident Management")
+    end
```

Comment:
This logic is already implemented in `Team.get_visible_teams`
```suggestion
    Team.get_visible_teams(current_user)
```

Reply:
@slorek done!